android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/FormConfigurationJsonWizardFormActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.vijay.jsonwizard.activities;
+
+import com.vijay.jsonwizard.delegates.JsonWizardFormDelegate;
+
+import org.json.JSONException;
+
+public class FormConfigurationJsonWizardFormActivity extends FormConfigurationJsonFormActivity{
+
+    private final JsonWizardFormDelegate delegate = new JsonWizardFormDelegate();
+
+    @Override
+    public void initializeFormFragment() {
```

Comment:
I see that you `synchronized` the same method for the `JsonWizardFormActivity`. Should this apply for this method?

Reply:
yeah. it should